assets/javascripts/test_result.js

```diff
@@ -868,12 +872,8 @@ function renderInvestigationTab(response) {
             var logDetailsDiv = document.createElement('div');
             logDetailsDiv.id = 'collapse' + key + i;
             logDetailsDiv.className = 'collapse';
-            stats = gitstats[i].split('\n')[0];
-            spanElem.innerHTML = stats;
-            logDetailsDiv.innerHTML = gitstats[i]
-              .split('\n')
-              .slice(1, gitstats[0].length - 1)
-              .join('\n');
+            spanElem.innerHTML = gitstats[i]['link'] + ' ' + gitstats[i]['msg'];
```

Comment:
```suggestion
            spanElem.innerHTML = gitstats[i].link + ' ' + gitstats[i].msg;
```
The same counts for other occurrences.